src/testing/dataFiles/pkginstall/debian/6/apache2-php.ini

```diff
@@ -1,5 +1,7 @@
 [PHP]
+; SPDX-FileCopyrightText: © Fossology contributors
 
+; SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Better license will be FSFAP for this file

Reply:
Uodated.